runtime/services/catalog/migrator/sources/sources.go

```diff
@@ -297,8 +300,17 @@ func source(connector string, src *runtimev1.Source) (drivers.Source, error) {
 			Query:    query,
 			Database: db,
 		}, nil
+	case "bigquery":
+		query, ok := props["query"].(string)
+		if !ok {
+			return nil, fmt.Errorf("property \"query\" is mandatory for connector \"bigquery\"")
+		}
```

Comment:
Could we make the field `sql` instead of `query`? It would be better compatible with the async reconcile parser

Reply:
Sure. I can make the same change for `MotherDuck` as well.